modules/gitops-release-notes-1-6-2.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-6-2_{context}"]
+= Release notes for {gitops-title} 1.6.2
+
+{gitops-title} 1.6.2 is now available on {product-title} 4.8, 4.9, 4.10 and 4.11.
+
+[id="fixed-issues-1-6-2_{context}"]
+== Fixed issues
+
+The following issue have been resolved in the current release:
+
+* Before this update, subscription health check was marked *degraded* for missing *InstallPlan* when more than 5 operators are installed in a project. This update fixes the issue. link:https://issues.redhat.com/browse/GITOPS-2018[GITOPS-2018]
```

Comment:
In this RN, you go from past tense to present tense. I think it should all be in past tense, so it'd be:

Before this update, subscription health check _was_ marked *degraded* for missing *InstallPlan* when more than 5 operators _were_ installed in a project.